web/client/actions/mediaEditor.js

```diff
@@ -135,3 +135,5 @@ export const importInLocal = ({resource, sourceType = SourceTypes.GEOSTORY, owne
     sourceType,
     owner
 });
+
+export const removeMedia = ({type: mediaType, owner = "geostory"}) => ({type: REMOVE_MEDIA, mediaType, owner});
```

Comment:
it's simpler like this to me
```suggestion
export const removeMedia = (mediaType, owner = "geostory") => ({type: REMOVE_MEDIA, mediaType, owner});
```

but this would require other changes where this action is called